src/cockpit/router.py

```diff
@@ -146,6 +146,9 @@ def __init__(self, routing_rules: List[RoutingRule]):
         self.no_endpoints = asyncio.Event()
         self.no_endpoints.set()  # at first there are no endpoints
 
+    def info(self) -> JsonObject:
+        return {}
```

Comment:
I think that's some leftovers from the previous approach?

Reply:
No.  This is the base implementation meant to be overridden from the bridge.  I'd use the override decorator but we don't have that on Python 3.6.  I'll add a comment.

Reply:
Added a docstring.